scripts/automated-patch-releases.mermaid

```diff
@@ -0,0 +1,11 @@
+---
+title: How CRI-O Automated Patch Releases Work
+---
+
+flowchart TD
+  A[Start: Patch Release Workflow] --> B[Create Version Bump Intent PRs]
+  B --> C[Maintainers Review PRs]
+  C -->|Decides Against Merges| E[End]
```

Comment:
It's not the end, the workflow will run again and rebase which should we reflect here.

Reply:
So after it rebases, the maintainers have to review it again right?